src/v0/destinations/ga4/utils.js

```diff
@@ -474,11 +474,21 @@ const basicValidation = (event) => {
  * @param {*} message
  * @returns
  */
-const getGA4ClientId = (message, Config) => {
+const getGA4ClientId = (message, Config, destType) => {
   let clientId;
 
   if (isHybridModeEnabled(Config)) {
-    const integrationsObj = getIntegrationsObj(message, 'ga4');
+    let integrationsObj = {};
```

Comment:
```suggestion
    const integrationsObj = getIntegrationsObj(message, destType);
```
won't this suffice ?